Sources/DataPipeline/CIODataPipeline.swift

```diff
@@ -0,0 +1,167 @@
+import CioInternalCommon
+import Foundation
+import Segment
+
+public enum CIODataPipeline {
+    private static var analytics: Analytics = .init(configuration: Configuration.defaultConfiguration)
```

Comment:
`analytics` object can't be optional, as `Segment` always expect it be non nullable. `CIODataPipeline` will be initialized from `CustomerIOImplementation` on SDK launch so there shouldn't be a case where this would be nullable.